xcode/Subconscious/Shared/Services/DataService.swift

```diff
@@ -663,13 +663,7 @@ actor DataService {
         let identity = try await noosphere.identity()
         var feed: [EntryStub] = []
         for entry in try self.database.listFeed(owner: identity) {
-            let author = try await Func.run {
-                if entry.address.isOurs {
-                    return try await self.userProfile.loadOurProfileFromMemo()
-                } else {
-                    return try await self.userProfile.buildUserProfile(address: entry.address)
-                }
-            }
+            let author = try await self.userProfile.identifyUser(did: entry.did, address: entry.address, context: nil)
```

Comment:
We could go further here and retrieve the user profile as part of `listFeed` using a `JOIN`. 

This would result in a lot of duplicate data in the query results but would resolve in one sweep. I didn't think this was necessary yet because, while we do call `identifyUser` for each entry, the in-memory cache in `UserProfileService` will make these cheap when building the feed.